codespell_lib/data/dictionary.txt

```diff
@@ -14870,6 +14873,7 @@ indicaite->indicate
 indicat->indicate
 indicateds->indicated, indicates,
 indicatee->indicates, indicated,
+indicats->indicates
```

Comment:
Perhaps this should also suggest `indicate`, as it's only one-letter different from `indicats`?